### PR TITLE
Ability to use piwikproprovider in server components - bugfix

### PR DESCRIPTION
**Pull request by lysy-vlc**
This PR fixes [this issue](https://community.piwik.pro/t/error-with-nextjs-setup/602/7?u=anthonybartczak)

### CODE REVIEW

File: example/app/layout.tsx [hunk 1]
```diff
@@ -1,9 +1,8 @@
-'use client'
```
Comment (auto200) [inline example/app/layout.tsx:1]: Issue: update examples in readme (remove `'use client'`)

File: src/index.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+'use client'
```
Comment (auto200) [inline src/index.tsx:1]: Thought: presence of `'use client'` here seems not to change anything for me

Comment (lysy-vlc) [reply src/index.tsx]: Double checked that and You're right. Reverted that

File: example/app/layout.tsx (hunk 1 above)
Comment (lysy-vlc) [reply example/app/layout.tsx:1]: done

**Review by danieltwork — APPROVED** [review]

**Pull request merged**